docs/feature_unicode.md

```diff
@@ -75,56 +75,55 @@ void qk_ucis_symbol_fallback (void) { // falls back to manual unicode entry
 }
 ```
 
-## Unicode Input methods
+## Input Modes
 
-Unicode input in QMK works by inputting a sequence of characters to the OS,
-sort of like macro. Unfortunately, each OS has different ideas on how Unicode is input.
+Unicode input in QMK works by inputting a sequence of characters to the OS, sort of like a macro. Unfortunately, the way this is done differs for each platform, so a corresponding input mode has to be set.
 
-You can set the input method at any time.  You can do this by using a keycode here.  The Input method is listed next to the keycode for reference. 
-
-|Key                       |Aliases  |Input Method  |Description                                        |
```

Comment:
I think having the keycodes at the top, since it's more visible, and ideally what should be used.

Reply:
Yeah, that occurred to me too, but you have to explain what input modes are first (and list which ones are available) before you tell people how to use them.

Also, as far as just _using_ Unicode is concerned (i.e. setting your preferred input mode if you just use one OS — the majority of use cases), keycodes still aren't ideal (and won't be the recommended method once `UNICODE_SELECTED_MODES` is added). They'll still be used for switching between modes, but I think simple usage should be prioritized.

Reply:
Yeah, after re-reading it, I agree